### DISCUSSION TITLE
Not able to use loading delay with display property

### DISCUSSION THREAD

**Discussion by SoNiice** [livewire/livewire#4811] [7. Bugs (archived)]
Hey!

I've tried multiple solutions posted by different people using both, the loading delay as well as the display property for the livewire-loading element. Are there any plans on fixing this? Is this even a bug or does it simply not work? Tried in v2.10.4 - Seems like it was working before in some versions, but I didn't downgrade to give it a try myself.

**Comment by joshhanley** [comment]
@SoNiice can you share some code of what you mean, so we can see exactly what isn't working? Thanks!

**Comment by SoNiice** [comment]
@joshhanley According to old entries on github and different forums this was working just fine in older versions of livewire.

```html
<div wire:loading.flex wire:loading.delay>
        <div style="display: flex; justify-content: center; align-items: center; background-color: black; position: fixed; top: 0; left: 0; z-index: 9999; width: 100%; height: 100%; opacity: .75;">
            <div class="fa fa-spinner fa-pulse fa-3x"></div>
        </div>
    </div>
```

It sadly doesn't anymore, I also found no working workaround to get both, the style and the delay working together.
